templates/message/message.html

```diff
@@ -0,0 +1,34 @@
+
+{% load static %}
+<link rel="stylesheet" href="{% static 'styles/message.css' %}">
+<div aria-live="polite" aria-atomic="true" style="position: relative; min-height: 200px;">
+  <!-- Position it -->
+  <div style="position: fixed; top: 90px; right: 28px;z-index:2">
+
```

Comment:
![image](https://user-images.githubusercontent.com/40713373/86973465-458a8480-c17d-11ea-9806-b0cdcce35660.png)
it's pretty neat

Reply:
This is cool, but please make sure to remove the top white-blank bar... this can't be merged like that.